chainer/training/extensions/plot_report.py

```diff
@@ -51,17 +51,20 @@ class PlotReport(extension.Extension):
     .. warning::
 
         If your environment needs to specify a backend of matplotlib
-        explicitly, please call ``matplotlib.use`` before importing Chainer.
-        For example:
+        explicitly, please call ``matplotlib.use`` before calling
+        ``trainer.run``. For example:
 
         .. code-block:: python
 
             import matplotlib
             matplotlib.use('Agg')
 
-            import chainer
+            trainer.extend(
+                extensions.PlotReport(['main/loss', 'validation/main/loss'],
+                                      'epoch', file_name='loss.png'))
+            trainer.run()
 
-        Then, once ``chainer.training.extensions`` is imported,
+        Then, once ``trainer.run`` is called,
```

Comment:
I think "once one of instances of this extension is called" is more accurate (and I believe is not complicated)